plugins/vue/src/index.js

```diff
@@ -0,0 +1,112 @@
+import VueTypes from 'vue-types';
```

Comment:
Why is not built-in props validation used?
https://vuejs.org/v2/guide/components.html#Prop-Validation